README.md

```diff
@@ -20,20 +20,20 @@ Wikipedia les décris ainsi
 
 > En informatique, et plus particulièrement en développement logiciel, un patron de conception (plus souvent appelé design pattern) est un arrangement caractéristique de modules, reconnu comme bonne pratique en réponse à un problème de conception d'un logiciel. Il décrit une solution standard, utilisable dans la conception de différents logiciels.
 
-⚠️ Be Careful
------------------
-- Design patterns are not a silver bullet to all your problems.
+⚠️ Faites Attention
+-------------------
+- Les design patterns ne sont pas des solutions-miracles à tous vos problèmes.
 - Do not try to force them; bad things are supposed to happen, if done so. Keep in mind that design patterns are solutions **to** problems, not solutions **finding** problems; so don't overthink.
```

Comment:
I translated line 26 but I have some doubts about the reliability of the translation.
Here is the translation :
`N'essayez pas de les forcer; des mauvaises choses sont censées se produire, si c'est fait. Gardez à l'esprit que les design patterns sont des solutions à des problèmes et non pas des solutions à des problèmes trouver; alors n'y pensez pas trop.`